src/library_gl.js

```diff
@@ -1507,17 +1513,24 @@ var LibraryGL = {
         usage = 0x88E8; // GL_DYNAMIC_DRAW
         break;
     }
+#endif
```

Comment:
This block was actually missing `#if LEGACY_GL_EMULATION`, coming from an old commit https://github.com/kripken/emscripten/commit/d2a3ea8ca82533ac4dae07605aa56dee8bee9558.